Information-system (Информационни системи)/Писмен (practical)/07.2016/task-3.hs

```diff
@@ -0,0 +1,20 @@
+-- Задача 3. Нека са дадени следните изрази на езиците Haskell и Scheme. Да се посочи
+-- каква е оценката на изразите. Изберете само един от двата езика за решението на задачата и
+-- напишете името му в даденото за целта поле. Точки за задачата се дават само за избрания от
+-- Вас език.
+-- Избран език: Haskell:
+
+-- 1. [x:[x]|x<­[[1,2],[3,4]]]
+-- Оценка: [[[1, 2], [1, 2]], [[3, 4], [3, 4]]]
+
+-- 2. [map(f5)[1,2,3]|f<­[(+),(­),(*)]]
```

Comment:
Spaces between map arguments would be needed

`f 5` instead of `f5`

`f<-[...` instead of `f<[...`

`(-)` instead of `()`